### ISSUE TITLE
Issue with reading .mp3 file with ID3v2.4 and ID3v1 tags

### ISSUE THREAD

**Issue by icidasset** [aadsm/jsmediatags#32]
When I try to read it with jsmediatags I get an empty `tags` object, but when I use an app like [this one](https://www.nightbirdsevolve.com/meta/) it reads the tags just fine.

The tags are in the beginning of the file, so I don't think it's related to #23 .
You can find the file here: http://icidasset-private.s3.amazonaws.com/01-skream-exothermic_reaction-bnp.mp3

**Comment by aadsm** [comment]
In this situation v2.4 tags will be preferred over v1. The problem is that these v2.4 tags use unsynchronisation which is not yet supported by this library.

You may reverse the order of tag priority by:
```javascript
jsmediatags.Config.removeTagReader(ID3v2TagReader);
jsmediatags.Config.addTagReader(ID3v2TagReader);
```

**Comment by icidasset** [comment]
Thanks! 🙏☺️

**Comment by aadsm** [comment]
@icidasset had a few hours this morning and just implemented unsynchronisation support. It's available starting version 3.3.0.

**Comment by icidasset** [comment]
@aadsm Awesome! 😱 Thanks again!!
